### PR TITLE
start using fake_tpdb

### PR DESCRIPTION
**Pull request by 4c0d3r**

### CODE REVIEW

File: src/templates/render/failedFiles.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% if data %}
-    <table id="{{ active_page }}" class="table table-sm text-start align-middle">
```
Comment (DirtyRacer1337) [inline src/templates/render/failedFiles.html:2]: This render also uses for queue page

Comment (4c0d3r) [reply src/templates/render/failedFiles.html:2]: it seems to work well in both with the new id

Comment (4c0d3r) [reply src/templates/render/failedFiles.html:2]: the problem with using the active_page is that the id changes when the web_root is set.

Comment (DirtyRacer1337) [reply src/templates/render/failedFiles.html:2]: still stays `failed` or `queue`, it's can't change because of web_root

**Pull request merged**